functions

```diff
@@ -49,9 +49,10 @@ service_create() {
     ROOTPASSWORD="$SERVICE_ROOT_PASSWORD"
     dokku_log_warn "Specified root password may not be as secure as the auto-generated root password"
   fi
-  echo "$ROOTPASSWORD" > "$SERVICE_ROOT/ROOTPASSWORD"
-  echo "$PASSWORD" > "$SERVICE_ROOT/PASSWORD"
-  chmod 640 "$SERVICE_ROOT/ROOTPASSWORD" "$SERVICE_ROOT/PASSWORD"
+  mkdir -p "$SERVICE_ROOT/auth"
+  echo "$ROOTPASSWORD" > "$SERVICE_ROOT/auth/root"
+  echo "$PASSWORD" > "$SERVICE_ROOT/auth/mariadb"
```

Comment:
`$SERVICE_ROOT/auth/mariadb` duplicates what's on line 17, I'd put it into a var to prevent code becoming diverse